### ISSUE TITLE
Special chars from ~ not working

### ISSUE THREAD

**Issue by vitormmatos** [telegramdesktop/tdesktop#27215]
### Steps to reproduce

1. Click on a chat to write a message (saved messages for example)
2. Type the '~' char following 'a' or 'o'

Obs: My keyboard layout is: English (US, intl., with dead keys)

### Expected behaviour

ã õ

### Actual behaviour

a o

### Operating system

Arch Linux, Gnome, x64

### Version of Telegram Desktop

4.12.2

### Installation source

Flatpak

**Comment by ilya-fedin** [comment]
Avoid flatpak if you need that
https://github.com/flatpak/flatpak/issues/5585

**Comment by vitormmatos** [comment]
> Avoid flatpak if you need that [flatpak/flatpak#5585](https://github.com/flatpak/flatpak/issues/5585)

What version would you recommend?

@Aokromes , how is this issue closed?
Is flatpak not a official supported build?

**Comment by ilya-fedin** [comment]
This is not a tdesktop bug, as you can see it's a flatpak bug that is reproducible in any sandboxed application

**Comment by vitormmatos** [comment]
> This is not a tdesktop bug, as you can see it's a flatpak bug that is reproducible in any sandboxed application

I get it now, sorry for the confusion, will try building from the source.
Thanks :)

**Comment by ilya-fedin** [comment]
Ah, wait, looks like I mis-read your issue. It doesn't look like https://github.com/flatpak/flatpak/issues/5585 yet it seems to be a duplicate of #27200 which is closed as being reproducible. You have to find what you're missing in steps to reproduce if you want the issue to be re-opened.

**Comment by vitormmatos** [comment]
> Ah, wait, looks like I mis-read your issue. It doesn't look like [flatpak/flatpak#5585](https://github.com/flatpak/flatpak/issues/5585) yet it seems to be a duplicate of #27200 which is closed as being unreproducible. You have to find what you're missing in steps to reproduce if you want the issue to be re-opened.

It's not a duplicate, the one before was downloaded from AUR, this from flatpak.

I tried right now to download the one from Linux 64 bit binary, same error.
Seems to be indeed, unfortunately,  an error in tdesktop

**Comment by ilya-fedin** [comment]
It's a duplicate, you're reporting the same issue. Other people are unable to reproduce it as you can see.

**Comment by ilya-fedin** [comment]
Personally I can't verify your issue because you haven't answered the question I asked you in the previous issue

**Comment by vitormmatos** [comment]
> It's a duplicate, you're reporting the same issue. Other people are unable to reproduce it as you can see.

I did not see many comments saying that they can't reproduce this but, but I agree with you that this is indeed the same issue.

I tried 3 different sources, AUR, flatpak, and now the Linux Binary, which was downloaded from this exact repo.

Not trying to be pricky here, I'm grateful for all the work done by collaborators in FOSS softwares, but something weird is going on.

Thanks
